test/e2e-go/features/stateproofs/stateproofs_test.go

```diff
@@ -226,6 +215,79 @@ func TestStateProofOverlappingKeys(t *testing.T) {
 	r.Equalf(int(consensusParams.StateProofInterval*expectedNumberOfStateProofs), int(lastStateProofBlock.Round()), "the expected last state proof block wasn't the one that was observed")
 }
 
+func TestStateProofMessageCommitmentVerification(t *testing.T) {
+	partitiontest.PartitionTest(t)
+	defer fixtures.ShutdownSynchronizedTest(t)
+
+	r := require.New(fixtures.SynchronizedTest(t))
+
+	configurableConsensus := make(config.ConsensusProtocols)
+	consensusVersion := protocol.ConsensusVersion("test-fast-stateproofs")
+	consensusParams := getDefaultStateProofConsensusParams()
+	configurableConsensus[consensusVersion] = consensusParams
+
+	var fixture fixtures.RestClientFixture
+	fixture.SetConsensus(configurableConsensus)
+	fixture.Setup(t, filepath.Join("nettemplates", "StateProof.json"))
+	defer fixture.Shutdown()
+
+	libgoalClient := fixture.LibGoalClient
+
+	restClient, err := fixture.NC.AlgodClient()
+	r.NoError(err)
+
+	var startRound = uint64(1)
+	var nextStateProofRound = uint64(0)
+	var firstStateProofRound = 2 * consensusParams.StateProofInterval
+
+	t.Logf("Waiting for state proof to be committed.")
```

Comment:
I think we can remove this log. since this test is quite short we can do without logs.

Reply:
This means that when running the test manually you get no feedback at all. Are we sure that's OK?